packages/mock-addon/.github/workflows/release.yml

```diff
@@ -12,10 +12,10 @@ jobs:
       - name: Prepare repository
         run: git fetch --unshallow --tags
 
-      - name: Use Node.js 14.x
-        uses: actions/setup-node@v1
+      - name: Use Node.js 18.x
```

Comment:
Minor, but since we're updating things, we might as well choose Node.js version 20, since that is the latest LTS version released.